stanford/suclass/group_vars/all

```diff
@@ -15,6 +17,7 @@ EDXAPP_LMS_BASE: 'suclass.stanford.edu'
 EDXAPP_LMS_BASE_SCHEME: 'https'
 EDXAPP_LMS_NGINX_PORT: 80
 EDXAPP_PREVIEW_LMS_BASE: "preview.{{ EDXAPP_LMS_SITE_NAME }}"
+EDXAPP_REGISTRATION_EXTENSION_FORM: ''
```

Comment:
@dcadams I went ahead and disabled this anyway on SUclass;
then we don't need to worry about SUclass migrations
and the extra table/data.